coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
The biggest problem with defining `Root` as default aspect is that a bear must be able to generate a result for every aspect is supports. And as @adhikasp correctly mentioned, `Root` would mean that the bear supports all aspects. Interpreting `Root` in any other way would break aspects definition consistency, since it is very beneficial to define a whole aspects sub-tree supported by a bear, like mentioned in #3977